go/database/mpt/rlp/rlp_test.go

```diff
@@ -12,82 +12,270 @@ package rlp
 
 import (
 	"bytes"
+	"fmt"
 	"math/big"
 	"testing"
 
 	"github.com/Fantom-foundation/Carmen/go/common"
 )
 
 func TestEncoding_EncodeStrings(t *testing.T) {
+	testWithRlpStrings(t, func(t *testing.T, rlp []byte, item String) {
```

Comment:
nit: rename "testWithRlpString" => "forEachRlpStringTest" ? Same for the others below.

Reply:
ok, renamed